src/locale/misc/updates/en.yml

```diff
@@ -14,7 +14,7 @@ journeyinstance:
   removeassignee: '{actor} unassigned {assignee}'
   removesubject: '{actor} removed {subject}'
   update:
-    summary_text: '{actor} edited the summary'
+    summary: '{actor} edited the summary'
```

Comment:
Nice catch! I made this change on the backend recently but forgot to change it on the frontend. I believe this means the `eslint-disable` at the top of this file can also be removed.